models/Transactional/LedgerPosting.ts

```diff
@@ -90,12 +90,30 @@ export class LedgerPosting {
       return map[account];
     }
 
+    // Timezone inconsistency fix (very ugly code for now)
+    const entryDateTime = this.refDoc.date as string | Date;
+    let dateTimeValue: Date;
+    if (typeof entryDateTime === 'string' || entryDateTime instanceof String) {
```

Comment:
Isn't that equivalent?

Reply:
It *should be*, yet as we saw with this DateTime fun, there are some weird errors that pop up. By forcing a check I want to make sure that it *is* as expected as I really don't want to revisit this error later.